.eslintrc.cjs

```diff
@@ -0,0 +1,51 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true
+  },
+  extends: [
+    'plugin:react/recommended',
+    'standard',
```

Comment:
this does not seem to be working. js code is not formatted in `standard` style.

Reply:
fixed

Reply:
@buraktt 
can you check if current changes fix this eslint configuration problem
thanks